src/frontend/scss/objects/_form.scss

```diff
@@ -1,12 +1,11 @@
-//
-// Form objects
-//
-
```

Comment:
Why remove this header ?

Reply:
i removed it when i moved it.
then didn't put it back.